src/main/java/io/fabric8/maven/docker/access/UrlBuilder.java

```diff
@@ -70,13 +70,19 @@ public String inspectContainer(String containerId) {
         return u("containers/%s/json", containerId)
                 .build();
     }
-    
+
     public String listContainers(int limit) {
         return u("containers/json")
                 .p("limit", limit)
                 .build();
     }
 
+    public String listAllContainers() {
+        return u("containers/json")
+                .p("all", 0)
```

Comment:
no worries, I already did some refactorings, will push my changes in an own branch soon.